docs/modules/ROOT/partials/proc-registry-quickstart-sr.adoc

```diff
@@ -1,22 +1,27 @@
 [id="registry-quickstart"]
 = Quickstart {registry} deployment
 
-To quickly create a new {registry} deployment, use the embedded Infinispan storage option,
-which does not require an external storage to be configured as a prerequisite.
+To create a new {registry} deployment, use the SQL storage option.
+This requires an external PostgreSQL storage to be configured as a prerequisite.
 
 .Prerequisites
 * Ensure that the {operator} is already installed.
+* You have a PostgreSQL database reachable from your OpenShift cluster.
 
 .Procedure
 * Create an `ApicurioRegistry` custom resource (CR) in the same namespace that the Operator is deployed:
 +
 [source,bash]
 ----
-oc apply -f ./examples/apicurioregistry_infinispan_cr.yaml -n service-registry
+oc project "$NAMESPACE"
+oc apply -f ./examples/apicurioregistry_sql_cr.yaml
 ----
 +
-.Example CR for Infinispan storage
+.Example CR for SQL storage
```

Comment:
I'm missing here some warning about having to update the database connection string in the CR